update(taxo): add super admin app (#1731)
bonitasoft/bonita-doc: modules/ROOT/pages/super-administrator-application-overview.adoc

= Bonita Super Administrator Application overview

> :description: This pages describes the Bonita _Super Administrator Application_: its values, how to access it, and how to customize it. +

Reviewer: The "the" sounds weird to me but maybe it is OK. It is just a suggestion. feel free to ignore
```suggestion
:description: This pages describes Bonita's _Super Administrator Application_: its values, how to access it, and how to customize it. +
```

Author: We should either capitalize without "the" or not capitalize with "the". You are right. I am making the change.

{description}

== Values

This application is the tool needed to:

* Bootstrap a Bonita Runtime in Non-Production and Production environments, for example, xref:first-steps-after-setup#_create_a_bonita_administrator_profile[creating the administrators]. +
* Pause BPM Services to perform maintenance operations, among which the update of the Business Data Model (and the related database migration).

It is only available to the user with the xref:special-users.adoc#_technical_user[Technical User] credentials. +

== Access during Development and Testing phases

To access the application:

. Open Bonita Studio
. Click on the *Applications* icon of the coolbar.

The default application is the Application Directory. You will be logged in as the default user of the deployed organization. +
This default user can be changed by opening the organization editor. +
By default, in our provided ACME organization, the default user is Walter Bates, logged as "walter.bates", with password "bpm". +

. Log out

> . Log back in with the Technical User credentials. In Bonita Studio, use username: `install` and password `install`. 

Reviewer: ```suggestion
. Log back in with the Technical User credentials. In Bonita Studio, use the username: `install` and password `install`. 
```

. In the Application Directory, select _Bonita Super Administrator Application_

You are now logged in Bonita Super Administrator Application.

To access its content: 

. From Bonita Super Administrator Application menu, click on "Applications"
. On the "Bonita Super Administrator Application" row, click on the "..." _View application details_ icon 

== Status
This application is core to Bonita Platform. It is embedded in the Runtime and cannot be downloaded from Bonita Studio nor be part of your automation project. +

> This is the reason why you will not see it in the studio Project Explorer. +

Reviewer: ```suggestion
This is the reason why you will not see it in Bonita Studio's Project Explorer. +
```

Author: Same here, both are fine: capitalized with no "the" or not capitalized, without "Bonita" with "the" : 
- Bonita Studio
- the studio

= Bonita Super Administrator Application overview
:description: This pages describes Bonita _Super Administrator Application_: its values, how to access it, and how to customize it. +

{description}

== Values

This application is only available to the user with the xref:special-users.adoc#_technical_user[Technical User] credentials. +
It is the tool needed to:

> * Bootstrap a Bonita Runtime in Non-Production and Production environments, for example, xref:first-steps-after-setup#_create_a_bonita_administrator_profile[creating the administrators]. +
> * Pause BPM Services to perform maintenance operations, among which the update of the Business Data Model (and the related database migration).

Reviewer: ```suggestion
* Bootstrap a Bonita Runtime in Non-Production and Production environments, for example, xref:first-steps-after-setup#_create_a_bonita_administrator_profile[creating the administrators]
* Pause BPM Services to perform maintenance operations, among which the update of the Business Data Model (and the related database migration)
```

= Bonita Super Administrator Application overview
:description: This pages describes Bonita _Super Administrator Application_: its values, how to access it, and how to customize it. +

{description}

== Values

This application is only available to the user with the xref:special-users.adoc#_technical_user[Technical User] credentials. +
It is the tool needed to:

* Bootstrap a Bonita Runtime in Non-Production and Production environments, for example, xref:first-steps-after-setup#_create_a_bonita_administrator_profile[creating the administrators]. +

> * Pause BPM Services to perform maintenance operations, among which the update of the Business Data Model (and the related database migration).

Reviewer: ```suggestion
* Bootstrap a Bonita Runtime in Non-Production and Production environments, for example, xref:first-steps-after-setup#_create_a_bonita_administrator_profile[creating the administrators]
* Pause BPM Services to perform maintenance operations, among which the update of the Business Data Model (and the related database migration)
```
